TRACONBoundaries.geojson

```diff
@@ -465,6 +465,7 @@
99444444],[21.3897,55.2828],[21.399999722222,55.2661],[21.4314,55.251899444444],[21.4525,55.214399722222],[21.479199722222,55.206699722222],[21.5039,55.1933],[21.517499722222,55.191699722222],[21.5517,55.199199722222],[21.596699722222,55.1878],[21.640299722222,55.184699444444],[21.654399722222,55.178899444444],[21.6828,55.158599166667],[21.715599722222,55.146099722222],[21.727499722222,55.1347],[21.7661,55.1244],[21.812799722222,55.118299722222],[21.8519,55.094999444444],[21.863099722222,55.0919],[21.944199722222,55.0778],[21.9889,55.088299722222],[22.009699722222,55.0878],[22.018899722222,55.083899166667],[22.023299722222,55.077199722222],[22.0178,55.056899444444],[22.0206,55.049399444444],[22.0408,55.0339],[22.0619,55.0275],[22.098099722222,55.0281],[22.121099722222,55.0375],[22.140599722222,55.053899444444],[22.197199722222,55.0617],[22.295799722222,55.066899722222],[22.3411,55.064399444444],[22.37,55.0581],[22.406099722222,55.058599444444],[22.448599722222,55.0508],[22.48,55.054999444444],[22.553599722222,55.0758],[22.574699444444,55.077199722222],[22.602199722222,55.0722],[22.602199722222,55.0467],[22.636099722222,54.985799722222],[22.640799722222,54.9664],[22.659199722222,54.958599722222],[22.732199722222,54.962499444444],[22.739999722222,54.944699722222],[22.7706,54.938899722222],[22.754999722222,54.9247],[22.7556,54.9206],[22.7814,54.917499722222],[22.835,54.901699444444],[22.849399722222,54.8875],[22.861666666667,54.855277777778],[22.867499722222,54.831099444444],[22.867799722222,54.8106],[22.846099444444,54.78],[22.8378,54.775799722222],[22.8136,54.7719],[22.7831,54.743099444444],[22.769999722222,54.736899444444],[22.7433,54.731099722222],[22.7406,54.7278],[22.7453,54.717199444444],[22.742499722222,54.7053],[22.7192,54.6914],[22.727199722222,54.647799722222],[22.745,54.636944444444],[22.7225,54.621699722222],[22.698299722222,54.589399722222],[22.7053,54.5628],[22.6944,54.5275],[22.6944,54.4733],[22.713599722222,54.4397],[22.7408,54.4194],[22.7572,54.393299722222],[22.771399444444,54.383099444444],[22.786110555556,54.363888333333],[22.763610833333,54.358888611111],[22.664166111111,54.3508325],[22.354166666667,54.3433325],[22.143055555556,54.334443611111],[22.012777777778,54.336943888889],[21.952777777778,54.331666388889],[21.411943611111,54.325832222222],[21.224721666667,54.333055277778],[21.140555833333,54.34],[21.074999722222,54.348888611111],[20.974166944444,54.355555833333],[20.724721666667,54.358888611111],[20.641666944444,54.366666666667],[20.580832777778,54.377499722222],[20.4327775,54.381666944444],[20.301388611111,54.397777222222],[20.090555555556,54.418610277778],[20.043888888889,54.410277777778],[20.004166388889,54.419166666667],[20,54.419519166667],[19.879443333333,54.429721111111],[19.848055833333,54.429999444444],[19.683332777778,54.449166388889],[19.641666666667,54.458333333333],[19.406666666667,54.603333333333],[19.406666666667,54.603333333333],[17.55,55.85],[18.018333333333,56.095],[20.643333333333,55.373333333333]]]]}},
```

Comment:
For the most part looks good, however there is significant overlap of the border with Gdansk with a lot of outliars that should be cleaned up, reference the image below.

![Screen Shot 2022-09-22 at 12 42 15](https://user-images.githubusercontent.com/967239/191726531-d9f60ddb-dcd3-4b47-b366-2e4fe70fab01.png)